src/controllers/sold.land.filter.controller.ts

```diff
@@ -2,14 +2,18 @@ import { NextFunction, Request, Response } from "express";
 
 import { logger } from "../utils/logger";
 import * as config from "../config";
+import { ApplicationData } from "../model";
+import { getApplicationData, setExtraData } from "../utils/application.data";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `GET ${config.SOLD_LAND_FILTER_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
 
     return res.render(config.SOLD_LAND_FILTER_PAGE, {
       backLinkUrl: config.LANDING_URL,
-      templateName: config.SOLD_LAND_FILTER_PAGE
+      templateName: config.SOLD_LAND_FILTER_PAGE,
+      has_sold_land: appData.has_sold_land
```

Comment:
Wondering why variable name format here looks different - underscores not CamelCase?